spec/std/crypto/bcrypt_spec.cr

```diff
@@ -47,4 +47,12 @@ describe "Bcrypt" do
     hash = Crypto::Bcrypt.digest("secret", 5)
     Crypto::Bcrypt.verify("Secret", hash).should be_false
   end
-end
\ No newline at end of file
+
+  it "check the speed" do
+    time = Time.now
+    Crypto::Bcrypt.digest("secret")
+    time = Time.now - time
+    (time.to_f < 0.07).should be_true # at least faster than ruby ;)
+  end
```

Comment:
I don't know, such specs tend to fail randomly IME, due to load issues while you run them or other flukes.